BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/LocationUtil.java

```diff
@@ -0,0 +1,62 @@
+package com.nexenio.bleindoorpositioning.location;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by leon on 29.03.18.
+ */
+
+public class LocationUtil {
+
+    public static Location meanLocationFromLast(List<Location> locationList, long amount, TimeUnit timeUnit) {
+        List<Location> matchingLocations = getLocationsFromLast(locationList, amount, timeUnit);
+        return meanLocation(matchingLocations);
+    }
+
+    public static Location meanLocation(List<Location> locationList) {
```

Comment:
Rename to `calculateMeanLocation` to stay consistent (see `AdvertisingPacketUtil`)